app/assets/javascripts/pdfjs_viewer/viewer.js.erb

```diff
@@ -7141,12 +7141,11 @@ var PDFViewerApplication = {
 };
 window.PDFView = PDFViewerApplication; // obsolete name, using it as an alias
 
+var HOSTED_VIEWER_ORIGINS = <%= ENV.fetch("PDFJS_VIEWER_ORIGINS", "").split(",").inspect %>;
 
-var HOSTED_VIEWER_ORIGINS = ['null',
-  'http://mozilla.github.io', 'https://mozilla.github.io'];
 function validateFileURL(file) {
   try {
-    var viewerOrigin = new URL(window.location.href).origin || 'null';
+    var viewerOrigin = window.location.origin;
```

Comment:
@Elektron1c97 also we don't want to change that code really. It's property of the PDF.js project. Every update we make needs to reapply these changes.